app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -609,5 +616,15 @@ def generic_remote_options_hash(repo_params)
       end
       generic_remote_options
     end
+
+    def check_import_parameters
+      repo_type = RepositoryTypeManager.find_by_content_type(params[:repository][:content_type])
```

Comment:
Since repository id is being passed in as a parameter, could we just look up the repository type with that?   Katello::Repository.find(id).repository_type

Then we wouldn't need a new method

Reply:
Oh that's a good point. Thanks!